src/style.css

```diff
@@ -0,0 +1,123 @@
+body {
+  font-family: "Avenir-Book", sans-serif;
+  font-size: 16px;
+  margin: 8px;
+  display: block;
+}
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+.search-bar {
+  width: 100%;
+  margin: 20px auto 0;
+  display: flex;
+  justify-content: center;
+}
+
+.search-bar__large {
+  height: 70px;
+}
+
+.input-label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.input-label__large::after {
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.input-label::after {
+  background: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-size: contain;
+  position: absolute;
+  content: "";
+  display: block;
+}
+
+.input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgb(61 78 97 / 10%);
+  outline: none;
+}
+
+.input__large {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+::placeholder {
+  font-family: "Avenir-Book", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.search-bar:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
```

Comment:
these two cause an unwanted side-effect of double-border:
![unwanted side-effect](https://user-images.githubusercontent.com/39308717/159220603-8495a3e0-8031-445c-b2f3-fc21c6d9df71.png)
better not have it
```suggestion
```

Reply:
Okay Done. Thank you.